xml/System.ComponentModel.Design.Serialization/MemberRelationship.xml

```diff
@@ -175,7 +175,7 @@
       </Parameters>
       <Docs>
         <param name="other">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Infrastructure support to make this a first class struct</summary>
```

Comment:
This seems like an old change, but I'm not sure why we're capturing it just now. @gewarren did this assembly exist before in dotnet-api-docs?

Also, @dotnet/area-system-componentmodel can you please provide a more meaningful summary?

Here's the source: https://github.com/dotnet/runtime/blame/dbef8db7c4e1d20dab771f234441c91770f66a58/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/MemberRelationshipService.cs#L204

Reply:
I see there have been some recent modifications to the file, even though most of the `Equals` or `==` overloads have been there for a few years.

@stephentoub are we ok with keeping that summary?

https://github.com/dotnet/runtime/blame/dbef8db7c4e1d20dab771f234441c91770f66a58/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/MemberRelationshipService.cs#L221